bin/update-all-reports

```diff
@@ -16,29 +16,42 @@ BEGIN {
     require "$d/../setenv.pl";
 }
 
+use FixMyStreet::DB;
 use FixMyStreet::Script::UpdateAllReports;
-use File::Path ();
-use File::Slurp;
+use Path::Tiny;
 use Getopt::Long::Descriptive;
 use JSON::MaybeXS;
 
 my ($opt, $usage) = describe_options(
     '%c %o',
     [ 'table', "Output JSON for old table-style page." ],
-    [ 'areas', "Include area IDs in output JSON." ],
+    [ 'body=i', "Restrict results to a particular body (dashboard-style)." ],
+    [ 'all-bodies', "Generate set of results for all bodies." ],
+    [ 'areas', "Include area IDs in output JSON (table-style)." ],
     [ 'help', "print usage message and exit", { shortcircuit => 1 } ],
 );
 print($usage->text), exit if $opt->help;
 
 my ($data, $filename);
 if ($opt->table) {
     $data = FixMyStreet::Script::UpdateAllReports::generate($opt->areas);
-    $filename = 'all-reports.json';
+    output('all-reports', $data);
+} elsif ($opt->all_bodies) {
+    my $bodies = FixMyStreet::DB->resultset("Body")->search({ deleted => 0 });
+    while (my $body = $bodies->next) {
+        my $data = FixMyStreet::Script::UpdateAllReports::generate_dashboard($body->id);
+        output("all-reports-dashboard-" . $body->id, $data);
+    }
 } else {
-    $data = FixMyStreet::Script::UpdateAllReports::generate_dashboard();
-    $filename = 'all-reports-dashboard.json';
+    my $body = $opt->body || "";
+    $data = FixMyStreet::Script::UpdateAllReports::generate_dashboard($body);
+    $body = "-$body" if $body;
+    output("all-reports-dashboard$body", $data);
 }
```

Comment:
There's a chance that this will end up with weird data if the body doesn't exist but then again if the body doesn't exist the weird data shouldn't be used so I guess that's fine. Might be nice to issue a warning though?